cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -357,6 +357,90 @@ def test_schedulable_memory_validator(schedulable_memory, ec2memory, instance_ty
             "Instance types listed under Compute Resource TestComputeResource must have the same inference "
             "accelerator manufacturer",
         ),
+        # Instance Types should have the same EFA support status if EFA is enabled
```

Comment:
add the test cases when there is no failure (e.g EFA enabled and all instances support EFA, EFA disabled and all instances don't support EFA)